airbyte-integrations/connectors/source-klaviyo/source_klaviyo/manifest.yaml

```diff
@@ -279,6 +293,8 @@ definitions:
         request_parameters:
           "include": "metric"
           "fields[metric]": "name"
+          "filter": "greater-than({{ parameters.cursor_field }},{{ stream_interval.start_time }})"
```

Comment:
@maxi297 - To confirm, should this be greater-than, or greater-or-equal?

I see instances of both in the deleted custom code file.

Reply:
My understand is that streams using `KlaviyoCheckpointDatetimeBasedCursor` would have `greater-or-equal` (see https://github.com/airbytehq/airbyte/pull/48391/files#diff-e318ea8c2956167d366ea080f5e4dbb993adfd22cdee41b4b7d1f7dcd36f2d3dL55) while the others would only have `greater_than`. This logic doesn't make sense to me and I would prefer to use `greater-or-equal` (better have duplication than missing records. 

@btkcodedev can we validate if `greater-or-equal` is supported for those streams and if so, update the filters  like @aaronsteers mentioned?

Reply:
All four changed streams support greater-than and greater-or-equal, for filtering, 
We could wait for this PR to merge and 
Wait for the reflected changes to be successful and make another PR after as it is a certified one.